oozie/__init__.py

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2017 "Shopify inc." All rights reserved.
```

Comment:
The overall package and project name is prefaced with `py` to distinguish it from the Oozie project, but it seems redundant to preface the module name with `py` since we'll be importing this in Python files.